test/tests/selftest/pbs_managers_operators_ext.py

```diff
@@ -50,13 +50,27 @@ def test_managers_unset_setup(self):
         after test setUp run
         """
         runas = ROOT_USER
+        manager_usr_str = str(MGR_USER) + '@*'
         current_usr = pwd.getpwuid(os.getuid())[0]
-        attr = {ATTR_managers: current_usr + '@*'}
-        self.server.expect(SERVER, attrib=attr)
+        current_usr_str = str(current_usr) + '@*'
+        svr_mgr = str(self.server.status(SERVER, 'managers'))
+        self.assertIn(str(manager_usr_str), svr_mgr)
+        self.assertIn(str(current_usr_str), svr_mgr)
+
         mgr_user1 = str(TEST_USER)
         mgr_user2 = str(TEST_USER1)
         a = {ATTR_managers: (INCR, mgr_user1 + '@*,' + mgr_user2 + '@*')}
         self.server.manager(MGR_CMD_SET, SERVER, a, runas=runas)
         self.logger.info("Calling test setUp:")
         TestSelf.setUp(self)
-        self.server.expect(SERVER, attrib=attr)
+
+        svr_mgr = str(self.server.status(SERVER, 'managers'))
+        self.assertIn(str(manager_usr_str), svr_mgr)
+        self.assertIn(str(current_usr_str), svr_mgr)
+
+    def test_default_oper(self):
+        """
+        Check that default operator user is set on PTL setup
+        """
+        svr_opr = str(self.server.status(SERVER, 'operators'))
+        self.assertIn(str(OPER_USER) + '@*', svr_opr)
```

Comment:
Again, this doesn't check that there aren't other operator users in the 'operators' attribute. Get the value and match it exactly.